importer/importer.py

```diff
@@ -43,8 +43,17 @@ def __init__(self, countryname, languagename):
         self.file_content = self.load_mapping_file(countryname, languagename)
 
 
-def make_query(specific_table):
-    return ('select DISTINCT *  from `{}`').format(specific_table)
+def make_query(specific_table, offset):
+    """
+    Generate a query to retrieve data from database.
+
+    :param specific_table: Name of table to retrieve data from.
+    :param offset: Optional offset to start from.
+    """
+    query = 'select DISTINCT *  from `{}`'.format(specific_table)
+    if isinstance(offset, int):
+        query = query + " LIMIT {}, 18446744073709551615;".format(str(offset))
```

Comment:
Apparently, in MySQL the only way to use OFFSET is as an argument of LIMIT, and then you have to specify a limit as well. [It's even in the documentation](https://dev.mysql.com/doc/refman/5.7/en/select.html) so, seriously, yeah (ctrl + f 18446744073709551615)